source/tutorial/getting-started.txt

```diff
@@ -252,8 +252,8 @@ To set up your own deployment:
    * - MongoDB Atlas Free Tier Cluster
     
      - MongoDB Atlas is a fast, easy, and free way to get started with
-       MongoDB. Follow the :ref:`create-free-tier-manual` tutorial to
-       get started with MongoDB Atlas.
+       MongoDB. To learn more, see the
```

Comment:
We follow the style "To learn more, see ____", but if you prefer "For more information, see" just let me know.

Reply:
This is fine.